extensions/panache/mongodb-panache/runtime/src/main/java/io/quarkus/mongodb/panache/jackson/ObjectMapperProducer.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.mongodb.panache.jackson;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+import javax.inject.Singleton;
+
+import org.bson.types.ObjectId;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+/**
+ * This ObjectMapperProducer will produce an ObjectMapper that will override the default one from resteasy-jackson.
```

Comment:
Yes, I'm not very happy with it. 
For the moment it doesn't breaks things because the object mapper from resteasy is empty but we must revisit this and allow other extensions to customize the ObjectMapper used by resteasy.
I tried to access the ObjectMapper from Arc in a BuildStep without success ...

Reply:
Perhaps @gsmet will have an idea? We could always accept this as a first step and open an issue for the limitation.

Reply:
I'll open a followup issue for this one.

Reply:
Maybe stupid idea, but couldn't we scan the index looking for serializers and register them in the `resteasy-jackson` extension?